### PR TITLE
Changing the EPSG3857.PROJECTION array assignment and adding urn:ogc:…

### PR DESCRIPTION
**Pull request by wnordmann**
adding projection urn:ocg:crs:EPSG::3857 
while removing a duplicate EPSG:102113
Changed epsg3857.projection to match spsg4326.projection

### CODE REVIEW

File: src/ol/proj/epsg3857.js [hunk 1]
```diff
@@ -64,30 +64,20 @@ ol.proj.EPSG3857.WORLD_EXTENT = [-180, -85, 180, 85];
...
  * @const
  * @type {Array.<ol.proj.Projection>}
  */
-ol.proj.EPSG3857.PROJECTIONS = ol.proj.EPSG3857.CODES.map(function(code) {
-  return new ol.proj.EPSG3857.Projection_(code);
-});
+ol.proj.EPSG3857.PROJECTIONS = [
+  new ol.proj.EPSG3857.Projection_('EPSG:3857'),
+  new ol.proj.EPSG3857.Projection_('EPSG:102100'),
+  new ol.proj.EPSG3857.Projection_('EPSG:102113'),
```
Comment (ahocevar) [inline src/ol/proj/epsg3857.js]: You removed the most important alternative code, 'EPSG:900913'. Please add it back.

Comment (wnordmann) [inline src/ol/proj/epsg3857.js]: Copy paste error, fixed

**Comment by bartvde** [comment]
Which data contained ```urn:ogc:def:crs:EPSG:3857```

Isn't that an invalid URN?

http://docs.opengeospatial.org/is/12-063r5/12-063r5.html

**Comment by wnordmann** [comment]
Agreed, but the original code was using the urn:ogc:def:crs:EPSG:3857 I
didn't want to break other code

**Comment by bartvde** [comment]
Am I reading the diff incorrectly then? I only see ```urn:ogc:def:crs:EPSG:3857``` added

**Comment by bartvde** [comment]
```urn:ogc:def:crs:EPSG::3857``` which was there is a valid URN

**Comment by wnordmann** [comment]
You are right, urn:ogc:def:crs:EPSG::3857 is valid and
urn:ogc:def:crs:EPSG:3857 is invalid, somewhere yesterday I got something
mixed up.  Sorry for the confusion.

**Comment by ahocevar** [comment]
@wnordmann: Can you please submit a follow-up pull request that removes the invalid URN?

**Pull request merged**